frappe/charts
# Fix bound listener

###### Explanation About What Code Achieves:

  - Fixes #97

###### Screenshots/GIFs:

  - Screenshot

###### Steps To Test:

  - Construct a test page with more than one frappe-charts on it
  - Remove the charts
  - Resize the window
  - Should not see any error in JS console about can't `removeChild`

###### TODOs:

  - None
